python_scripts/02_numerical_pipeline_scaling.py

```diff
@@ -23,30 +23,29 @@
 # * an example of preprocessing, namely the **scaling numerical variables**;
 # * using a scikit-learn **pipeline** to chain preprocessing and model
 #   training;
-# * assessed the performance of our model via **cross-validation** instead of
+# * assessing the performance of our model via **cross-validation** instead of
 #   a single train-test split.
 #
 # ## Data preparation
 #
-# Let's charge the full adult census dataset and split the target from the
-# actual data used for modeling.
+# First, let's charge the full adult census dataset.
 
 # %%
 import pandas as pd
 
 df = pd.read_csv("../datasets/adult-census.csv")
 
 # %% [markdown]
-# We will first split our dataset to have the target separated from the data
-# used to train our predictive model.
+# We will now drop the target from the data we will use to train our
```

Comment:
```suggestion
# We will now drop the target from the data that we will use to train our
```

Reply:
Both forms are valid, but I think not using it lightens the sentence, especially on the pronunciation (*the data we* vs *the data that*) :)